### PR TITLE
Fix context menu no longer opening for hitobjects in timeline

### LINKED ISSUE #18470: Can't right click objects in the editor timeline
### Type

Game behaviour

### Bug description

You can't right click objects on the timeline in the editor. You could previously do this. (I think this has been a bug since last version?)

### Screenshots or videos

https://user-images.githubusercontent.com/68173946/170847283-9c4d3683-539a-4c69-9228-457130ad8386.mp4

### Version

2022.528.0

### Logs

[performance.log](https://github.com/ppy/osu/files/8792403/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8792404/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8792405/updater.log)
[database.log](https://github.com/ppy/osu/files/8792406/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/8792407/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/8792408/network.log)

### PR DESCRIPTION
**Pull request by frenzibyte**
- Closes #18470 

Working around this for now, since figuring out a proper solution would probably not be that easy.

### CODE REVIEW

**Comment by bdach** [comment]
I guess this works for the time being? If I were to suggest what the "ideal" solution here would be, it'd probably be an opt-in `Prioritised` flag [like the one `KeyBindingHandler` has](https://github.com/ppy/osu-framework/blob/2528319cb20dd3b5937ba4f745eca2dee2326b0d/osu.Framework/Input/Bindings/KeyBindingContainer.cs#L76-L79).

I can live with this workaround for now, but I'd want to see a second approval here before I'd be comfortable with merging.

**Review by bdach — APPROVED** [review]

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**
